news/2019-02-22-acknowledging-community-contributors.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: post
+title: "Acknowledging Community Contributors"
+date: 2019-02-22 00:00:00 +0000
+---
+
+At its very heart, osu! is almost entirely driven by its community. And at the heart of our community, the efforts of an outstanding few make themselves known. Today, we take a moment to reflect on some of these people, and award them the recognition they deserve.
+
+![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/banner.jpg)
+
+Perhaps the most prestigious of all profile badges awarded, the [community contributor badge](https://assets.ppy.sh/profile-badges/contributor.jpg) is expressly granted to those of the osu! community who have gone utterly above and beyond the norm in their enduring contribution to the well-being of the game and its community.
+
+In a first for many years, we are proud to announce the latest set of community contributors in what is the largest number of additions made since the initiative started back in 2013.
```

Comment:
This almost sounds like we're announcing community contributors for the first time in many years as opposed to us announcing its the largest number of additions in many years. Perhaps omitting the first part is a good idea.

Reply:
i fix